projects/kit/components/toast/toast.style.less

```diff
@@ -0,0 +1,37 @@
+@import '@taiga-ui/core/styles/taiga-ui-local';
+
+[tuiToast] {
+    --t-radius: 1rem;
+    --t-min-height: 3.5rem;
+    --t-padding: 0.75rem 1rem;
+
+    display: flex;
+    box-sizing: border-box;
+    gap: 0.5rem;
+    background: var(--tui-background-elevation-3);
+    min-block-size: var(--t-min-height);
+    border-radius: var(--t-radius);
+    padding: var(--t-padding);
+    border: 1px solid var(--tui-border-darkmode-only);
```

Comment:
We don't have such variable in open source

Reply:
fixed